internal/rbd/nodeserver.go

```diff
@@ -84,6 +92,10 @@ var (
 			Backport:     true,
 		}, // RHEL 8.2
 	}
+
```

Comment:
@nixpanic do we need this whitespace here?

Reply:
not sure if it is required, but splitting a multi-line variable with an empty line between the next multi-line comment looks clean to me

Reply:
so in many places we dont leave a newline before multiline comment and in some places like this we do. Thats what I am trying to understand.